smoke.test.ts

```diff
@@ -0,0 +1,75 @@
+require('selenium-webdriver/chrome');
+require('selenium-webdriver/firefox');
+require('chromedriver')
+
+import { BrowserHelper} from "./browser.helper";
+import { GetPageLocators } from "./page.objects";
+
+const rootURL = 'https://auction-app-1.herokuapp.com/';
+
+
+
+let browser = new BrowserHelper();
+let pageLocators = new GetPageLocators();
+
+
+describe('Smoke tests for Auction Application', () => {
+
+  describe('Tests - home page', () => {
+
+    it('Check Search option by inserting Monitor item', async () => {
+
+      await browser.goToPage(rootURL);
+      await browser.driver.sleep(5000);
+      await browser.enterDataAndClick(pageLocators.getHomePage().searchBox, "Monitor", pageLocators.getHomePage().searchBtn);
+      await browser.driver.sleep(500);
```

Comment:
we should consider to use some other type of inspect if element is visible or clickable, instead of using only driver.sleep command to slow load of a page; try to investigate about this part